docs/source/upgrade/upgrade-guide-from-monitoring-3.x-to-monitoring-4.y.rst

```diff
@@ -202,37 +201,34 @@ Typically, you need to back-fill the recording rules when you are using a long r
 and you upgraded to Scylla Monitoring 3.8 about three months ago.
 
 If you open the Overview dashboard and look at your entire retention time (in our example 1 year) you will see that while most of the graphs do
-show the data, the latency graphs have a missing period, in our example - from the entire year, the latency graph will only show the last three months.
-
-That nine months gap (12 months minus 3) is what we want to fill with back-filling.
+display the data, the latency graphs are missing a period of time.  The latency graph will only show the last three months in our example from the entire year. That nine months gap (12 months minus 3) is what we want to fill with back-filling.
 
 The point in time that the graphs start will be your back-filling end time. Check in the graph for the exact time.
 
 Backfilling Process
 -------------------
-backup
+Backup
 ^^^^^^
-If you have a long retention period you are using an external directory that holds the Prometheus data, back it up, in case
-If you have a long retention period, you are using an external directory that holds the Prometheus data back it up; if something goes wrong in the process, you can revert the process.
+Backup the external directory containing Prometheus data; if something goes wrong, you can revert the changes.
+
+To complete the process, you must restart Monitoring Stack at least once. You cannot complete the process without providing the path to the external directory with Prometheus data using the ``-d`` command line option. 
 
-To complete the process you will need to restart the monitoring stack at least once. If you are not using an external directory (The ``-d``
-command-line option) You cannot complete it.
 
 Restart the monitoring stack
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-You need to stop the monitoring stack and run the ``stat-all.sh`` command with an additional flag:
+You need to stop the monitoring stack and run the ``start-all.sh`` command with an additional flag (The 365d retention time used here is used only as an example):
```

Comment:
The additional flag is "--storage.tsdb.allow-overlapping-blocks" it should be clear

Reply:
Let's add the flag name:

You need to stop the monitoring stack and run the ``start-all.sh`` command with an additional flag `--storage.tsdb.allow-overlapping-blocks`. In the following command, the 365d retention time used is used as an example: